src/Compiler/Checking/TypeHierarchy.fs

```diff
@@ -139,6 +139,8 @@ let GetImmediateInterfacesOfMetadataType g amap m skipUnref ty (tcref: TyconRef)
 // IComparable<T> or IEquatable<T>. This is because whether they support these interfaces depend on their
 // constituent types, which may not yet be known in type inference.
 let rec GetImmediateInterfacesOfType skipUnref g amap m ty =
+    // TODO(vlza): Cache? Do we already do that via the importmap?
```

Comment:
That'd probably be a "no" - we don't want to cache immediate interfaces, they're quite cheap to get.